docs/source/conf.py

```diff
@@ -59,7 +57,6 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = []
 
-
```

Comment:
Removed the `snowflake_rtd_theme` part of `conf.py` from the Modin version of the docs, removed the entire `snowflake_rtd_theme` since it is no longer used.